sdk/metric/meter.go

```diff
@@ -436,6 +438,11 @@ func (r observer) ObserveFloat64(o metric.Float64Observable, v float64, opts ...
 		return
 	}
 
+	if oImpl.dropAggregation {
```

Comment:
This assumes only the drop aggregation is used for this instrument. There may be other measures that need to recorded here.

Reply:
Good point! I've changed it to only effect whether we log the error https://github.com/open-telemetry/opentelemetry-go/blob/875fca6bd0fb97b17f54db717a67b097adea2130/sdk/metric/meter.go#L478-L486